### PR TITLE
store/driver: migrate test-infra to testify

### LINKED ISSUE #27653: migrate test-infra to testify for `store/driver`

### PR DESCRIPTION
**Pull request by tisonkun**
Signed-off-by: tison <wander4096@gmail.com>

### What problem does this PR solve?

Issue Number: close #27653

### Release note

```release-note
None
```

### CODE REVIEW

File: store/driver/util_test.go [hunk 1]
```diff
@@ -1,82 +0,0 @@
...
-func TestT(t *testing.T) {
-	CustomVerboseFlag = true
-	tikv.EnableFailpoints()
-	TestingT(t)
-}
-
-var (
-	pdAddrs  = flag.String("pd-addrs", "127.0.0.1:2379", "pd addrs")
-	withTiKV = flag.Bool("with-tikv", false, "run tests with TiKV cluster started. (not use the mock server)")
-)
```
Comment (tisonkun) [inline store/driver/util_test.go:40]: @disksing @XuHuaiyu I don't see any where to config this variable, and thus remove them just respect the default.

Comment (disksing) [reply store/driver/util_test.go:40]: I think Ci environment will use it to test with a real tikv server.

Comment (tisonkun) [reply store/driver/util_test.go:40]: @zhouqiang-cl I remember that won't be one, at least for now. Please confirm.

Comment (tisonkun) [reply store/driver/util_test.go:40]: OK that may make sense, let me try to confirm it and if true, update the code.

Comment (tisonkun) [reply store/driver/util_test.go:40]: confirmed https://github.com/PingCAP-QE/ci/blob/bcb0c98ff3abfbf0a1e1ab8dc60ea280ef95d1ee/jenkins/pipelines/ci/tidb/tidb_ghpr_check_2.groovy#L132

Updating....

Comment (tisonkun) [reply store/driver/util_test.go:40]: Updated. @disksing PTAL.

**Review by disksing — APPROVED** [review]

**Review by lcwangchao — APPROVED** [review]

**Comment by tisonkun** [comment]
/assign @disksing 

Could you please help on merging this PR?

**Comment by disksing** [comment]
CI is too slow. will try at midnight.

**Pull request merged**